### PR TITLE
Bug/pi name missing 246

### PR DESCRIPTION
**Pull request by NWalker4483**
Adds an endpoint to return a list of users associated with a study

### CODE REVIEW

File: crc/api.yml [hunk 1]
```diff
@@ -1565,6 +1589,17 @@ components:
           type: string
           x-nullable: true
           example: "27b-6-1212"
+    StudyAssociate:
```
Comment (danfunk) [inline crc/api.yml:1592]: This is nice, you are really thinking through the API.

File: crc/api/study.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 from crc.api.common import ApiError, ApiErrorSchema
 from crc.models.protocol_builder import ProtocolBuilderStatus
 from crc.models.study import Study, StudyEvent, StudyEventType, StudyModel, StudySchema, StudyForUpdateSchema, \
-    StudyStatus
+    StudyStatus, StudyAssociatedSchema
```
Comment (danfunk) [inline crc/api/study.py:10]: Great that we have a Schema for StudyAssociated but did you use it?

File: crc/services/study_service.py [hunk 3]
```diff
@@ -121,14 +122,14 @@ def get_study_associate(study_id = None, uid=None):
             raise ApiError('uid not specified','A valid uva uid is required for this function')
 
         if uid == study.user_uid:
-            return {'uid': ownerid, 'role': 'owner', 'send_email': True, 'access': True}
+            return {'uid': uid, 'role': 'owner', 'send_email': True, 'access': True}
```
Comment (danfunk) [inline crc/services/study_service.py:125]: Here is where we should use an object or a schema.  Seems like the full model should be returned, then passed through the schema.

**Review by danfunk — CHANGES_REQUESTED** [review]
The API is tight, and I like that you created a good schema for Study Associate, but you don't seem to use the schema.  We are also missing tests around this.

File: crc/api/study.py (hunk 2 above)
Comment (NWalker4483) [reply crc/api/study.py:10]: It was breaking the test_email_cc and a few other seemingly random tests so I was planning to come back to it to figure out once the main feature was working

Comment (NWalker4483) [reply crc/api/study.py:10]: I can draft this PR and figure that out now tho

**Pull request merged**